llpc/patch/llpcPatchDescriptorLoad.cpp

```diff
@@ -438,8 +440,22 @@ Value* PatchDescriptorLoad::LoadDescriptor(
                 "",
                 pInsertPoint);
 
-            auto pDescOffset = ConstantInt::get(Type::getInt32Ty(*m_pContext), descOffset);
-
+            llvm::Value* pDescOffset = nullptr;
+            if (!m_pPipelineState->GetBuilderContext()->BuildingRelocatableElf())
+            {
+                pDescOffset = ConstantInt::get(Type::getInt32Ty(*m_pContext), descOffset);
+            }
+            else
+            {
+                IRBuilder<> builder(*m_pContext);
+                builder.SetInsertPoint(pInsertPoint);
+                SmallVector<Value*, 16> args;
+                args.push_back(ConstantInt::get(Type::getInt32Ty(*m_pContext), descSet));
+                args.push_back(ConstantInt::get(Type::getInt32Ty(*m_pContext), binding));
```

Comment:
builder.getInt32, plus feeding the args directly to CreateIntrinsic, as in {arg0, arg1}.